linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/oval/shared.xml

```diff
@@ -1,4 +1,9 @@
 <def-group>
+  {{% if 'ubuntu' in product %}}
```

Comment:
`/sbin` is a symlink to `/usr/sbin` 
this change is not necessary and also your change is currently breaking the escaping of `/` and you also didn't fix the bash remediation and the rule is not passing anymore when using bash. So please drop this commit

Reply:
I agree with this, similar to the other comments, DISA Benchmark will search specifically for `/sbin`, but should not be a problem. I'm dropping this commit. Thanks for the feedback!